### DISCUSSION TITLE
Cant understand the meaning of the message

### DISCUSSION THREAD

**Discussion by Edidemiyamba** [termux/termux-packages#11804] [General]
OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: open /dev/ptmx: no space left on device: unknown
Connection to d269493efd6f.54be720d.alx-cod.online closed.
~ $ pkg install root
Checking availability of current mirror: bad
Testing the available mirrors:
[*] https://packages-cf.termux.org/apt/termux-main: bad
[*] https://deb.kcubeterm.me/termux-main: bad
[*] https://termux.mentality.rip/termux-main: bad
[*] https://grimler.se/termux-packages-24: bad
[*] https://termux.librehat.com/apt/termux-main: bad
Using fallback mirror: https://packages-cf.termux.org/apt/termux-main
Hit:1 https://packages-cf.termux.dev/apt/termux-main stable InRelease
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
48 packages can be upgraded. Run 'apt list --upgradable' to see them.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package root
~ $

**Comment by aicynide** [comment]
run `termux-change-repo` and select a working mirror

**Comment by ChiefMikeK** [comment]
1. **"root"** is not a program that you can download or install.
2. Termux does not provide any way to root your device.
3. You need to update 48 other programs. 
4. run `pkg up` every few days.
5. sometimes all the mirrors are `bad` just retry `pkg up` later.
6. Try toggle your wifi off and on (my wifi gets congested a lot) :neckbeard:

**Comment by OrdinaryEnder** [comment]
Did u mean proot

**Reply by ChiefMikeK** [reply]
> Did u mean proot

OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: open /dev/ptmx: no space left on device: unknown
Connection to d269493efd6f.54be720d.alx-cod.online closed.
~ $ pkg install root

**Reply by OrdinaryEnder** [reply]
bruh

**Reply by OrdinaryEnder** [reply]
`proot` not root

**Reply by OrdinaryEnder** [reply]
Also ur have no space lol

**Reply by ChiefMikeK** [reply]
if /dev is on a readonly mount point android will say it is full
